app/src/main/res/values/colors.xml

```diff
@@ -31,8 +31,8 @@
     <color name="colorOnPrimaryDark">@color/white</color>
 
     <!-- NOTE: Must be in sync with the colors in the WidgetUtils class. -->
-    <color name="colorSecondary">#9e9e9e</color>
-    <color name="colorSecondaryLight">#cfcfcf</color>
```

Comment:
Why did you change the color?

Reply:
Because I added margin to the layout. Doing so, the old background color didn't really fit it.

Reply:
OK